packages/components/src/components/radio-group/radio-group.stories.ts

```diff
@@ -41,280 +48,79 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * The sd-radio in all possible combinations of `orientation` and `size`.
- */
-
-export const Orientation = {
-  parameters: { controls: { exclude: ['orientation', 'size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'orientation' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the disabled attribute to disable a input radio. Clicks will be suppressed until the disabled state is removed
+ * Use the `size` attribute to change the size.
  */
 
-export const Disabled = {
-  name: 'Disabled x Size',
-  parameters: { controls: { exclude: ['size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: [
-          {
-            type: 'slot',
-            name: 'default',
-            title: 'disabled',
-            values: [
-              {
-                value:
-                  '<sd-radio value="1" disabled>Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'true'
-              },
-              {
-                value:
-                  '<sd-radio value="1">Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'false'
-              }
-            ]
-          }
-        ],
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group name="radio-group" size="lg">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+
+      <sd-radio-group name="radio-group" size="sm">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+    </div>
+  `
```

Comment:
Please "check" the first <sd-radio>.
<img width="583" alt="image" src="https://github.com/user-attachments/assets/0cc79f3f-db30-4dc3-a480-657a64cab276">